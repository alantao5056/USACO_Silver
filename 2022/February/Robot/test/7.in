40
-7 60
73 47
28 61
80 99
-20 -1
7 0
-99 -86
24 4
-22 -5
-44 -57
10 22
43 57
29 40
53 85
-91 -68
-11 12
21 48
-57 -86
-38 -69
-96 -66
75 63
85 95
-35 -61
-36 -3
-63 -34
57 78
-53 -32
-45 -66
26 31
36 6
-22 -32
-25 -58
-39 -17
-24 -33
12 9
66 75
-9 18
-46 -77
-16 0
-68 -59
36 67
