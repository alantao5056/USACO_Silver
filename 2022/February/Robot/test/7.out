0
0
0
1
17
81
393
1502
4993
14724
38146
87629
179362
329607
544503
809879
1089959
1326217
1460774
1457235
1313425
1070930
787564
521616
309130
163747
77145
31875
11375
3552
935
184
35
5
0
0
0
0
0
0
